charts/generic-indexer/values.yaml

```diff
@@ -23,16 +25,19 @@ resources:
     cpu: 500m
     memory: 1Gi
 
-annotations: {}
-
-# HA Configuration
 strategy:
   type: RollingUpdate
   rollingUpdate:
     maxUnavailable: 3
     maxSurge: 6
 
-# Pod disruption budget for HA
-podDisruptionBudget:
-  enabled: true
-  minAvailable: 2
+autoscaling:
+  enabled: false
+  minReplicas: 2
+  maxReplicas: 5
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+
+annotations: {}
+
+command: []
```

Comment:
Is this to set the command to `run` ? I think as the chart is specific to this binary, we should just hard code it in the chart so it doesn't cause confusion when using it, wdyt?